etc/profile-m-z/nodejs-common.profile

```diff
@@ -7,9 +7,26 @@ include nodejs-common.local
 # added by caller profile
 #include globals.local
 
+# Add the next line to your nodejs-common.local if you prefer to disable gatsby telemetry.
+#env GATSBY_TELEMETRY_DISABLED=1
```

Comment:
`env` should be placed in the mdwe,ro,rw,join-or-start block at the end.

Reply:
Done.